.github/workflows/regen-docs.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   regen:
-    if: github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch'
+    if: github.repository_owner == 'tidev' && (github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch')
```

Comment:
I _think_ this should work, not sure how fancy the logic you can do in these `if` blocks is

Reply:
We can only merge it to see oft it works, right?

Reply:
I think I could maybe test on my fork by setting the branch I created this PR from as the default branch. I'll give it a try

Reply:
It looks like this should be ok, [here's](https://github.com/ewanharris/titanium_mobile/actions/runs/2563755926) a run of `Regen Docs` against this branch which didn't run because of the check, and [here's](https://github.com/ewanharris/titanium_mobile/runs/7059991207) the master one that failed

Reply:
Then we should merge this. Are you okay with that?

Reply:
I'm good with merging! Not sure if @cb1kenobi wants a quick peek as he wrote these actions initially